lecture-notes/3-bld.md

```diff
@@ -939,17 +957,60 @@ factorial:                              ; @factorial
 
 You might notice that our chosen variable names are gone (no more `x` and we
 have these funky-looking `eax`, `edi`, `.LBB0_1`, etc), there is no more
-obvious math (`x * factorial (x - 1)` is also gone), and the instructions
-appear to be a bit more regular looking... more linear. These are the qualities
-that people normally ascribe to lower-level languages.
+obvious math (`x * factorial(x - 1)` is also gone), and the instructions appear
+to be a bit more regular looking... more linear. There is no `if` and there are
+no curly braces... instead, there are only jump instructions and labels. These
+are the qualities that people normally ascribe to lower-level languages.
```

Comment:
Can this whole paragraph be a footnote? It's not directly related to the thrust of our point, which is the difference between compilation and linking.

Reply:
I think I should go into more lurid detail for linking so that they can form some kind of (potentially vague) distinction between the two